package.json

```diff
@@ -15,7 +15,7 @@
     "build": "hardhat compile && tsc -p tsconfig.build.json",
     "plugin:size-contracts": "hardhat size-contracts",
     "plugin:upload-selectors": "hardhat upload-selectors",
-    "plugin:storage-layout": "hardhat check > layout/storage-layout.txt && yarn plugin:generate-storage-layout --source layout/storage-layout.txt",
+    "plugin:storage-layout": "hardhat check > logs/storage.txt && yarn plugin:generate-storage-layout --source logs/storage.txt",
```

Comment:
```suggestion
    "plugin:storage-layout": "hardhat check > logs/storage_layout.txt && yarn plugin:generate-storage-layout --source logs/storage_layout.txt",
```

Reply:
@nxqbao It is not the destination of the extracted file, but rather the temporary file generated by the hardhat-storage-layout package. This file will be deleted once the task is completed.